_documentation/04-packaging-drivers.md

```diff
@@ -89,12 +88,14 @@ ___
 * **description**
 <br>Description is used to provide a little more detail about your application. You could as many lines as you want in this field.
 
-        description: |
-            HP Printer Application is a PAPPL (Printer Application Framework) based printer application
-            to support hp printers.
-            PAPPL is a simple C-based framework/library for developing CUPS
-            Printer Applications, which are the recommended replacement for
-            printer drivers.
+    ```c
```

Comment:
Why are you putting the ```c here? The c here means to use syntax highlighting for C, but your code blocks are pieces of `snapcraft.yaml` which is not C. To avoid that unrelated words get highlighted, do not use the c in such code blocks.

Reply:
Apologies Till, I will correct this.